OpenRA.Mods.D2k/Traits/Render/WithCrumbleOverlay.cs

```diff
@@ -35,21 +35,34 @@ public class WithCrumbleOverlayInfo : ConditionalTraitInfo, Requires<RenderSprit
 
 	public class WithCrumbleOverlay : ConditionalTrait<WithCrumbleOverlayInfo>
 	{
+		readonly Animation overlay;
+		readonly RenderSprites renderSprites;
+		readonly WithCrumbleOverlayInfo info;
+		readonly AnimationWithOffset anim;
```

Comment:
Minor style nit: I'd order these in the order they're populated (info, renderSprites, overlay, anim).